packages/synapse-react-client/CONTRIBUTING.md

```diff
@@ -4,37 +4,35 @@
 
 ### Setup
```

Comment:
This doc is not well maintained. I saw some references to UMD that I wanted to remove, since we don't use UMD anymore. I made some other easy changes.